src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -175,14 +177,47 @@ public void assertNotBlank(AssertionInfo info, CharSequence actual) {
   }
 
   private boolean isBlank(CharSequence actual) {
-    if (actual == null || actual.length() == 0) return false;
+    return isNullOrEmpty(actual) || notNullSeqContainsOnlyWhitespaces(actual);
+  }
+
+  private boolean containsOnlyWhitespace(CharSequence actual) {
```

Comment:
rename to `containsOnlyWhitespaces` (`s` added)

Reply:
Done